packages/cli/src/commands/doctor/runAutomaticFix.ts

```diff
@@ -1,46 +1,49 @@
-// @flow
 import chalk from 'chalk';
-import ora from 'ora';
+import ora, {Ora} from 'ora';
 import {logger} from '@react-native-community/cli-tools';
-// $FlowFixMe - converted to TS
 import {HEALTHCHECK_TYPES} from './healthchecks';
-import type {EnvironmentInfo} from './types';
+import {EnvironmentInfo, HealthCheckCategoryResult} from './types';
 
-const AUTOMATIC_FIX_LEVELS = {
-  ALL_ISSUES: 'ALL_ISSUES',
-  ERRORS: 'ERRORS',
-  WARNINGS: 'WARNINGS',
-};
+export enum AUTOMATIC_FIX_LEVELS {
+  ALL_ISSUES = 'ALL_ISSUES',
+  ERRORS = 'ERRORS',
+  WARNINGS = 'WARNINGS',
+}
 
-export {AUTOMATIC_FIX_LEVELS};
-export default async ({
+interface RunAutomaticFixArgs {
```

Comment:
@thib92: small question, when do you use `type` and `interface`?

Reply:
There's a small difference between the two, but to be fair I usually use `type` because you can union them, type functions, etc. I'm not sure what the advantages of `interface` is though. I saw you used `interface` before so I used them where I could

Reply:
Cool, thanks for replying!